modules/soe_paragraph_image_cta/config/install/core.entity_view_display.paragraph.stanford_image_cta.default.yml

```diff
@@ -26,28 +26,21 @@ third_party_settings:
         - stanford_image_cta_image
       link:
         - stanford_image_cta_link
+_core:
```

Comment:
The _core: line and the default_config_hash should be removed from the installation configs.  They only go in the profile sync configs.